cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -3101,168 +3103,509 @@ func TestMutateGlobalPullSecret(t *testing.T) {
 					".dockerconfigjson": []byte(`{"key":"value"}`),
 				},
 			},
-			expectedErr: fmt.Errorf("failed to get token for registry.ci.openshift.org"),
+			expectedErr: fmt.Errorf("failed to get any token"),
 		},
 		{
 			name: "bad original",
 			secret: &coreapi.Secret{
 				Data: map[string][]byte{
 					".dockerconfigjson": []byte(`{
-	"auths": {
-		"a": {
-			"auth": "foo",
-			"email": "e"
-		},
-		"registry.ci.openshift.org": {
-			"auth": "cool"
-		},
-		"c": {
-			"auth": "bar",
-			"email": "g"
-		}
-	}
-}`),
+						"auths": {
+							"a": {
+								"auth": "foo",
+								"email": "e"
+							},
+							"registry.ci.openshift.org": {
+								"auth": "cool"
+							},
+							"c": {
+								"auth": "bar",
+								"email": "g"
+							}
+						}
+					}`),
 				},
 			},
 			expectedErr: fmt.Errorf("failed to parse the original secret: failed to get content from nil secret"),
 		},
 		{
-			name: "basic case: expired auth is replaced",
```

Comment:
Could we keep this case and rename it to "basic case: auth for app.ci is expired auth is replaced"

Reply:
I do not see this one addresse.